### DISCUSSION TITLE
关于225行的Gap effect，在学习中随机停止 10 秒

### DISCUSSION THREAD

**Discussion by HIbian** [zijie0/HumanSystemOptimization#5] [Q&A]
请问随机停止10秒怎么理解，可以理解为看看远方，或者站起来一下之类的吗？

**Comment by zijie0** [comment]
出处来自这个视频: https://youtu.be/2ekdc6jCu2E
大约1:56:35开始。原文是"randomly stop and do nothing"，所以按我理解就是短暂在原地发呆放空哈。
